communication/marketing-team/social-guidelines.md

```diff
@@ -26,6 +26,11 @@ Kubernetes Organization Members with access to account are:
 - Contributor/SIG blog posts
 - Events
 - KEPs
+  - KEP tweets *MUST* meet these conditions:
+  - Author AND at least half of the assignee/reviewer/approver (named approver) team (total of 3 people at minimum)
+    - Author and Name Approvers(s) consent must be recorded on the KEP
+  - Best practice would be to link to a discuss.kubernetes.io topic but ok if KEP folks want GitHub Issue
+  - Focus on approved KEPs; A good example is user feedback for features that are alpha
```

Comment:
We need to be clear about KEPs vs. PRs against a KEP:
- I would ignore PR assignees when it comes to giving consent, and go only by author/reviewer/approver listed in the KEP metadata (to counteract: I open a PR, brigade a bunch of my friends on it as assignees, claim I'm the Author of the PR, and the assignees are all saying it's ok to tweet)
- I would only use KEP metadata that was already in the repo instead of KEP metadata updated in the PR (to counteract: I open a PR changing all of the KEP metadata to my friends, and they all say it's OK to tweet)
- In the event that there is no KEP metadata in the repo, it's not OK to tweet about it
- There is no such thing as an "approved" KEP.  We probably mean KEPs that have `status: implementable` in their metadata (which, again, I would source from the repo, not the PR)
- e.g. a PR that moves a KEP to `status: implementable` is not ok to tweet about until it has merged

Reply:
I think it'd be useful to put this policy under a heading so it's easy to link to, eg: `git.k8s.io/community/communication/marketing-team/social-guidelines.md#keps`

Reply:
PTAL